charts/pega/templates/pega-environment-config.yaml

```diff
@@ -110,33 +110,36 @@ data:
   # will be retried once.
   CASSANDRA_CUSTOM_RETRY_POLICY: "{{ .Values.dds.customRetryPolicy }}"
-  # Enable the use of a customized retry policy for your Pega Platform deployment. After you enable this policy in your
-  # deployment configuration, the deployment retries Cassandra queries that timeout. Configure the number of retries
-  # using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. The default is 1, so if you
-  # do not specify a retry count, timed out queries are retried once.
-  # This property is for use with Pega Infinity '23 and lower.
+  # Enable the use of a customized retry policy for your Pega Platform deployment for Pega Platform ’23 and earlier
+  # releases. After you enable this policy in your deployment configuration, the deployment retries Cassandra queries
+  # that time out. Configure the number of retries using the dynamic system setting (DSS):
+  # dnode/cassandra_custom_retry_policy/retryCount. The default is 1, so if you do not specify a retry count, timed out
+  # queries are retried once.
   CASSANDRA_CUSTOM_RETRY_POLICY_ENABLED: "{{ .Values.dds.customRetryPolicyEnabled }}"
-  # Enable the use of a customized retry policy for your Pega Platform deployment. After you enable this policy in your
-  # deployment configuration, the deployment retries Cassandra queries that timeout. Configure the number of retries
-  # using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. The default is 1, so if you
-  # do not specify a retry count, timed out queries are retried once.
-  # This property is for use with Pega Infinity '24 and greater.
+  # Use this parameter in Pega Platform '24 and later instead of `customRetryPolicy`. Configure the number of retries
+  # using the `customRetryPolicyCount` property.
   CASSANDRA_CUSTOM_RETRY_POLICY_COUNT: "{{ .Values.dds.customRetryPolicyCount }}"
-  # Specify the number of retries when custom retry policy is enabled.
+  # Specify the number of retry attempts when `customRetryPolicyEnabled` is true. For Pega Platform '23 and earlier
+  # releases use the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount.
   CASSANDRA_SPECULATIVE_EXECUTION_POLICY: "{{ .Values.dds.speculativeExecutionPolicy }}"
-  # Enable the speculative execution policy for retrieving data from your Cassandra service. When enabled, the Pega
-  # Platform will send a query to multiple nodes in your Cassandra service and process the first query response. This
-  # provides lower perceived latencies for your deployment, but puts greater load on your Cassandra service.
-  # This property is for use with Pega Infinity '23 and lower.
+  # Enable the speculative execution policy for retrieving data from your Cassandra service for Pega Platform '23 and
+  # earlier releases. When enabled, the Pega Platform will send a query to multiple nodes in your Cassandra service and
```

Comment:
cleanup a little wording>>
When enabled, the Pega Platform will send a query to multiple nodes in your Cassandra service and process the first query response.
to
When enabled, Pega Platform sends a query to multiple nodes in your Cassandra service and processes the first response.